worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -268,8 +275,9 @@ class PushDataHandler extends BaseMessageHandler with Logging {
             peer.getReplicatePort)
           if (unavailablePeers.containsKey(peerWorker)) {
             pushData.body().release()
-            wrappedCallback.onFailure(
-              new Exception(s"Peer $peerWorker unavailable for $location!"))
+            callback.onFailure(
```

Comment:
wrappedCallback.onFailure will call ```workerSource.incCounter(WorkerSource.PushDataFailCount)```. If we change to callback here, we should also call incCounter here

Reply:
> wrappedCallback.onFailure will call `workerSource.incCounter(WorkerSource.PushDataFailCount)`. If we change to callback here, we should also call incCounter here

Done